apps/dashboard/src/modules/financial/components/vat/VatTable.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <DataTable
+    data-key="id"
+    lazy
+    :loading="isLoading"
+    :paginator="paginator"
+    :rows="rows"
+    :rows-per-page-options="[5, 10, 25, 50, 100]"
+    table-style="min-width: 50rem"
+    :total-records="totalRecords"
+    :value="rowValues"
+    @page="onPage($event)"
+  >
+    <Column field="id" :header="t('common.id')">
+      <template #body="slotProps">
+        <Skeleton v-if="isLoading" class="h-1rem my-1 surface-300 w-6" />
+        <span v-else>{{ slotProps.data.id }}</span>
+      </template>
+    </Column>
+    <Column field="name" :header="t('common.name')">
+      <template #body="slotProps">
+        <Skeleton v-if="isLoading" class="h-1rem my-1 surface-300 w-8" />
+        <span v-else>{{ slotProps.data.name }}</span>
+      </template>
+    </Column>
+    <Column v-if="hasPercentage" field="percentage" :header="t('modules.financial.invoice.vat.percentage')">
+      <template #body="slotProps">
+        <Skeleton v-if="isLoading" class="h-1rem my-1 surface-300 w-6" />
+        <span v-else>{{ slotProps.data.percentage }}%</span>
```

Comment:
I really do not like the raw `%`
```suggestion
        <span v-else>{{ slotProps.data.percentage + '%'  }}</span>
```